easybuild/easyconfigs/o/octopus/octopus-11.4-foss-2021a.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'ConfigureMake'
+
+name = 'octopus'
+version = '11.4'
+
+homepage = 'https://octopus-code.org/wiki/Main_Page'
+description = """Octopus is a scientific program aimed at the ab initio
+virtual experimentation on a hopefully ever-increasing range of system
+types. Electrons are described quantum-mechanically within
+density-functional theory (DFT), in its time-dependent form (TDDFT) when
+doing simulations in time. Nuclei are described classically as point
+particles. Electron-nucleus interaction is described within the
+pseudopotential approximation."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['http://octopus-code.org/down.php?file=%(version)s']
+sources = [SOURCE_TAR_GZ]
+patches = [
+    'octopus-11.4_fix_shebang_perl.patch',
+]
+checksums = [
+    '73bb872bff8165ddd8efc5b891f767cb3fe575b5a4b518416c834450a4492da7',  # octopus-11.4.tar.gz
+    'd16d72f5f0e1b33a45000100982a8f7ee93643f512d86df6f2fdcb960859d290',  # octopus-11.4_fix_shebang_perl.patch
+]
+
+builddependencies = [
+    ('Perl', '5.32.1'),
+]
+
+dependencies = [
+    ('libxc', '5.1.5'),
+    ('GSL', '2.7'),
+    ('netCDF', '4.8.0'),
+    ('netCDF-Fortran', '4.5.3'),
+    ('ETSF_IO', '1.0.4'),
+    ('futile', '1.8.3'),
+    ('PSolver', '1.8.3'),
+    ('CGAL', '4.14.3'),
+    ('likwid', '5.2.0'),
+    ('DFTB+', '21.1'),
+    ('libvdwxc', '0.4.0'),
+    ('NLopt', '2.7.0'),
+    ('libyaml', '0.2.5'),
+    ('ELPA', '2021.05.001'),
+    # ('BerkeleyGW', '3.0.1'),  # Problems with output/output.F90, keep disabled
+    # ('libgd', '2.3.1'),  # requires v2.2.x for configure to find it
+    # ('libpspio', '0.2.4'),  # Requires older version where libnames are libfpspio and not libpspiof
+]
```

Comment:
octopus requires METIS to be built with single precision, we don't have that (yet) so for now I skipped that.
And without external METIS we can't ass ParMETIS

Reply:
@akesandgren It is possible to compile Octopus with just ParMETIS (and I strongly advise it, otherwise performance for some calculations will suffer greatly).

Reply:
How? during configure it says that it MUST have external METIS to build with ParMETIS and refuses if there is no external METIS configured. And it requires single precision built METIS.

Reply:
It works because ParMETIS ships with METIS.